examples/complete/fixtures.us-west-1.tfvars

```
@@ -0,0 +1,7 @@
+region = "us-west-1"
```

```suggestion
region = "us-east-2"
```

and change the file name to `examples/complete/fixtures.us-east-2.tfvars`